### DISCUSSION TITLE
when vsphere install worker ignition fail

### DISCUSSION THREAD

**Discussion by Hyun-June-Choi** [okd-project/okd#1273] [Q&A]
**Describe the bug**
bootstrap, master ignition success, when turn on worker vm and ignition fail over and over

**Version**
Vsphere UPI openshift-install-linux-4.10.0-0.okd-2022-06-10-131327

**How reproducible**
100% in my environment

**Log bundle**
[log-bundle-20220703233218.tar.gz](https://github.com/openshift/okd/files/9036241/log-bundle-20220703233218.tar.gz)

**Comment by vrutkovs** [comment]
See #1260

**Reply by Hyun-June-Choi** [reply]
is this same issue? 
In case of UPI, is it okay to boot the bootstrap first then prepared master and worker at the same time?
